src/publish.js

```diff
@@ -53,10 +58,14 @@ module.exports = function strain() {
         .help();
     },
     handler: async (argv) => {
-      if (!executor) {
+      if (argv.remote) {
+        // eslint-disable-next-line global-require
+        const PublishCommand = require('./remotepublish.cmd'); // lazy load the handler to speed up execution time
+        executor = executor || new PublishCommand(makeLogger(argv));
+      } else {
         // eslint-disable-next-line global-require
         const StrainCommand = require('./publish.cmd'); // lazy load the handler to speed up execution time
```

Comment:
when we're at, this should probably be renamed, too:
```suggestion
        const PublishCommand = require('./publish.cmd'); // lazy load the handler to speed up execution time
```